src/gitlab/configuration-model.ts

```diff
@@ -212,6 +212,20 @@ export interface Service {
   readonly entrypoint?: string[];
   /** Full name of the image that should be used. It should contain the Registry part if needed.*/
   readonly name: string;
+  /** The pull policy that the runner uses to fetch the Docker image */
+  readonly pullPolicy?: PullPolicy[];
+  /** Additional environment variables that are passed exclusively to the service.. */
+  readonly variables?: Record<string, number | string>;
```

Comment:
```suggestion
  readonly variables?: Record<string, string>;
```
@ctasada thanks for the fixes! Looks like you've missed one.

Reply:
Good catch @mrgrain I applied the change and rebuild to update the API.md too.